packages/visualizations/src/components/Map/Svg/Choropleth.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+    import { geoEqualEarth } from 'd3-geo';
+    import {
+        mapKeyToColor,
+        LIGHT_GREY,
+        DEFAULT_COLORSSCALE,
+        EMPTY_FC,
+        getDataBounds,
+        colorShapes,
+    } from '../utils';
+    import Map from './Map.svelte';
+
+    import type { GeoJsonChoroplethOptions, ChoroplethDataValue } from '../types';
+
+    export let data: { value: ChoroplethDataValue[] };
+    export let options: GeoJsonChoroplethOptions;
+
+    $: ({ shapes, colorsScale = DEFAULT_COLORSSCALE, emptyValueColor = LIGHT_GREY } = options);
+    $: colorMapping = mapKeyToColor(
+        data.value,
+        getDataBounds(data.value),
+        colorsScale,
+        emptyValueColor
+    );
+    $: coloredShapes =
+        shapes?.geoJson && shapes?.type === 'geojson'
+            ? colorShapes({
+                  featureCollection: shapes.geoJson,
+                  colorMapping,
+              })
+            : EMPTY_FC;
+</script>
+
+<Map
+    projection={geoEqualEarth()}
```

Comment:
```suggestion
    projection={geoMercator()}
```

D3 proposes the same projection used in every map on the web (Mercator). When I try it, I get more similar shapes, whereas the `geoEqualEarth` seem skewed to me (especially from DROM-COM that seems really streched)


Mercator
<img width="663" alt="Screenshot 2022-10-17 at 15 25 27" src="https://user-images.githubusercontent.com/1935200/196189101-b86b2a2a-c1dc-411f-8e89-c5971554c915.png">

Equal
<img width="660" alt="Screenshot 2022-10-17 at 15 25 54" src="https://user-images.githubusercontent.com/1935200/196189105-04dfdce6-6946-430b-960a-85cb4217032f.png">

Reply:
Yep, that was the default projection, but geoMercator is the one used by Maplibre, so it makes more sense !